packages/dht/src/connection/ManagedConnection.ts

```diff
@@ -57,6 +57,9 @@ export class ManagedConnection extends EventEmitter<Events> {
     protected outgoingConnection?: IConnection
     protected incomingConnection?: IConnection
 
+    // TODO: Temporary debug variable, should be removed in the future.
+    private created = Date.now()
+
```

Comment:
Do we need both `stopped` and `closing` fields?

Reply:
Yeah could take a look at that in the future